packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -23,12 +23,19 @@
 
 .bpk-navigation-bar {
   position: relative;
-  display: flex;
+  display: grid;
```

Comment:
Attempted to layout with flex after removing some absolute positioning on the buttons but the desired spacing and handing of buttons being optional for different scenarios goes beyond what we can easily do with Flexbox. Grid can easily achieve the different cases we need.